streams/kr.m3u

```diff
@@ -489,5 +489,5 @@ http://necgokr2-724.acs.wecandeo.com/ms/2528/724/index.m3u8
 http://live.kytv.co.kr:8080/hls/stream.m3u8
 #EXTINF:-1 tvg-id="" status="error",KT Alpha Shopping (1080p)
 https://kr-streams.onrender.com/scraper?channel=438
-#EXTINF:-1 tvg-id="" status="online",Maeil Business Newspaper TV (매일경제 TV) (1080p)
+#EXTINF:-1 tvg-id="MBN.kr" status="online",Maeil Business Newspaper TV (매일경제 TV) (1080p)
```

Comment:
This stream does not match with other MBN streams. I created a PR (#9821) to resolve the conflict.